pkg/analysis/passes/legacyplatform/legacyplatform.go

```diff
@@ -27,6 +28,7 @@ var Analyzer = &analysis.Analyzer{
 type detector interface {
 	// Detect takes the content of a module.js file and returns true if the plugin is using a legacy platform (Angular).
 	Detect(moduleJs []byte) bool
+	GetPattern() string
```

Comment:
nit (usually you don't add `GetSomething`):

```suggestion
	Pattern() string
```